package.json

```diff
@@ -8,8 +8,12 @@
     "build:electron:pixi": "cross-env NODE_ENV=production --graphics=pixi --target=electron-renderer webpack --mode=production",
     "build:web": "webpack --target=web --mode=production",
     "build:web:pixi": "cross-env --graphics=pixi webpack --mode=production",
-    "electron": "concurrently \"cross-env NODE_ENV=development --target=electron-renderer webpack --watch\" \"cross-env NODE_ENV=development electron .\"",
-    "electron:pixi": "concurrently \"cross-env NODE_ENV=development --graphics=pixi --target=electron-renderer webpack --watch\" \"cross-env NODE_ENV=development electron .\"",
+    "delay": "delay 5",
+    "electron:dev": "cross-env NODE_ENV=development --target=electron-renderer webpack --watch",
```

Comment:
I wonder if there's a way to mark scripts that are not meant to be run directly by the game developer, but only by other scripts. I guess we are getting into the territory of grunt/gulp here.

Reply:
that's an interesting comment - I really agree. Something to think about, but I have no suggestion at the moment.